.config/dictionaries/technical.dic

```diff
@@ -282,6 +283,7 @@ RUSTFLAGS
 rustfmt
 RUSTSEC # RUSTSEC-XXXX-XXXX is id of RustSec Advisory Database
 rustup
+sufficients
```

Comment:
```suggestion
```
Nit typo, failing spellcheck CI. Please confirm if this addition is necessary for syntax to work in [frontend/apps/picasso/defi/polkadot/pallets/Assets/statemine.ts](https://github.com/ComposableFi/composable/pull/2217/files#diff-350510d2f5d2f6d66add1a00507db201affc4343f8b1af4247a6f14b25876833)